pkg/Library/CoreDesigner/DesignApp.js

```diff
@@ -168,7 +168,7 @@ export const rebuildObject = async (layer, id) => {
 };
 
 const connectionsByObjectId = (graph, id) => {
-  const prefix = `${id}$`;
+  const prefix = `${id}\$`;
```

Comment:
i'm just curious - does the escaping really matter? it's not a regex, just a string...

Reply:
The syntax highlighter in Chrome DevTools was confused by the non-escaped $ ...